ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml

```diff
@@ -6688,7 +6686,7 @@ periodics:
       secret:
         secretName: result-aggregator
 - agent: kubernetes
-  cluster: build06
+  cluster: build10
```

Comment:
Is this change from b06 to b10 expected @deepsm007 ?

Reply:
no it shouldn't be b10